Assets/Scripts/Tools/GraphArrangerTool.cs

```diff
@@ -17,6 +18,8 @@ public GraphArrangerTool(GraphArranger graphArranger, ToolPanelController toolPa
 
     public string ToolName => "Arrange";
 
+    private void SetToolgunModeText() => _toolPanelController.SetToolgunModeText($"{ToolName}\n{GetArrangerMode()}");
```

Comment:
I am not sure if I like passing more info in an extra line like this. I think better solution would be to have a seperate game object under the mode text. Changes to the toolgun panel will then be easily possible, the way it is now code changes would also be required.

Reply:
ok, I'll add another Text object